src/Conda.jl

```diff
@@ -128,13 +174,6 @@ function _installer_url()
     elseif Compat.Sys.islinux()
         res *= "Linux"
     elseif Compat.Sys.iswindows()
-        if MINICONDA_VERSION == "3"
-            # Quick fix for:
-            # * https://github.com/JuliaLang/IJulia.jl/issues/739
-            # * https://github.com/ContinuumIO/anaconda-issues/issues/10082
-            # * https://github.com/conda/conda/issues/7789
-            res = "https://repo.continuum.io/miniconda/Miniconda$(MINICONDA_VERSION)-4.5.4-"
-        end
```

Comment:
Seems like this PR could be done before #125.

Reply:
Well..., the id of this PR is 121 (< #124)

Joking aside, this PR needs a coordination with PyCall to use SetDllDirectory/AddDllDirectory and for IJulia to tweak setup `ENV["PATH"]` before launching Jupyter Notebook. So I still think it makes sense to release a quick fix with #124 first to help Windows users.

I don't know much about Windows so please feel free to close this PR and implement another solution. I re-opened this PR because I though I'd better cleanup my suboptimal solution.